File: transita-mais-backend-priv/transita-commons/src/main/java/br/com/texsistemas/transitamais/commons/api/dto/veiculo/RotaVeiculoPontoDTO.java
package br.com.texsistemas.transitamais.commons.api.dto.veiculo;

import java.util.UUID;

public record RotaVeiculoPontoDTO(
    UUID veiculoId,
    UUID horarioId,
    UUID pontoId,
    Integer ordem
)  {
}

File: mobile/app/src/main/java/br/com/texsistemas/transita/data/web/dto/RotaVeiculoPontoDTO.kt
package br.com.texsistemas.transita.data.web.dto

import kotlinx.serialization.Serializable

@Serializable
data class RotaVeiculoPontoDTO(
    val veiculoId: String,
    val horarioId: String,
    val pontoId: String,
    val ordem: Int
)
